### PR TITLE
fix(account): fund command additional feedback for sending transactions + check

### PR DESCRIPTION
**Pull request by realdreamer**
- Print request key and explorer url
- Improve help text after funding an account
- Before sending transactions to chain validate using `local` (for account fund and tx send commands)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206794459798420
  - https://app.asana.com/0/0/1206794459798425

### CODE REVIEW

File: packages/tools/kadena-cli/src/account/commands/accountFund.ts [hunk 1]
```diff
@@ -126,15 +127,40 @@ export const createAccountFundCommand = createCommand(
...
-    const loader = ora('Funding account...\n').start();
-
     const result = await fund(config);
-    assertCommandError(result, loader);
+    assertCommandError(result);
 
     log.info(
       log.color.green(
-        `"${accountConfig.name}" account funded with "${amount}" ${accountConfig.fungible} on chain ${chainId} in ${networkConfig.networkId} network.`,
+        `Explorer URL: ${networkConfig.networkExplorerUrl}\nRequest Key: ${result.data.requestKey}`,
```
Comment (barthuijgen) [inline packages/tools/kadena-cli/src/account/commands/accountFund.ts:135]: I think this url should be including the request key:

`https://explorer.chainweb.com/testnet/tx/${result.data.requestKey}`

File: packages/tools/kadena-cli/src/account/utils/createAndTransferFunds.ts [hunk 2]
```diff
@@ -78,15 +78,16 @@ export async function createAndTransferFund({
...
     }
 
-    const { submit, listen } = createClient(
+    const { submit, local } = createClient(
       `${networkConfig.networkHost}/chainweb/0.0/${networkConfig.networkId}/chain/${chainId}/pact`,
     );
 
-    const transactionDescriptor = await submit(signedTx);
+    // Validate the transaction locally before sending it to the network
+    await local(signedTx);
```
Comment (barthuijgen) [inline packages/tools/kadena-cli/src/account/utils/createAndTransferFunds.ts:86]: does `local()` throw if something is wrong? or should the result be actually validated?

Comment (realdreamer) [reply packages/tools/kadena-cli/src/account/utils/createAndTransferFunds.ts]: yes, it does throws error but for some use cases needs to be validated. so added that check as well.

File: packages/tools/kadena-cli/src/tx/commands/txSend.ts [hunk 3]
```diff
@@ -139,6 +139,11 @@ export const sendTransactionAction = async ({
 
       const client = getClient(details);
 
+      const localResponse = await client.local(command);
+      if (localResponse.result.status === 'failure') {
+        throw localResponse.result.error;
```
Comment (nillo) [inline packages/tools/kadena-cli/src/tx/commands/txSend.ts:144]: throw a command error (check)

**Review by barthuijgen — APPROVED** [review]

**Review by nillo — APPROVED** [review]

**Review by nillo — APPROVED** [review]

**Pull request merged**